R/git_tools.R

```diff
@@ -184,15 +184,26 @@ install_external_deps <- function(repo_dir, internal_pkg_deps, ...) {
 # function to get the remote name (e.g. origin) which matches
 # the url given in the staged.deps yaml file
 get_remote_name <- function(git_repo, repo_url) {
+
+  # remove the https:// and .git from repo_url
+  repo_url <- gsub("^.+://", "", repo_url, perl = TRUE)
+  repo_url <- gsub(".git$", "", repo_url)
```

Comment:
line 190 is needed only if we assume that staged.dependencies.tyaml might contain x/y.git
If we assume so, please combine two lines into one regexp
```suggestion
  repo_url <- gsub("^.+://|.git$", "", repo_url, perl = TRUE)
```